suite-native/atoms/src/Accordion.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
+import { Pressable, View } from 'react-native';
+import Animated, {
+    Easing,
+    useAnimatedStyle,
+    useSharedValue,
+    withTiming,
+} from 'react-native-reanimated';
+
+import { Icon } from '@trezor/icons';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+import { Box } from './Box';
+import { Text } from './Text';
+
+type AccordionProps = {
+    title: string;
+    content: string;
+};
+
+const triggerStyle = prepareNativeStyle(utils => ({
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    paddingVertical: utils.spacings.small,
+}));
+
+const ANIMATION_DURATION = 500;
+
+export const Accordion = ({ title, content }: AccordionProps) => {
+    const { applyStyle } = useNativeStyles();
+    const [isOpen, setIsOpen] = useState(false);
+    const height = useSharedValue<string | number>(0);
```

Comment:
```suggestion
    const height = useSharedValue(0);
```

height should be always a `number` which is deduced from the initial value `0`. No need for specifying valid types.

Reply:
https://github.com/trezor/trezor-suite/pull/7792/commits/c595ab010953f51589666bab9f8a90f3dfa4cc36